omod/src/main/webapp/template/localHeader.jsp

```diff
@@ -10,7 +10,14 @@
 		href="${pageContext.request.contextPath}/module/dhisconnector/configureServer.form"><spring:message
 				code="dhisconnector.configureServer" /></a>
 	</li>
-	
+
+	<li
+			<c:if test='<%= request.getRequestURI().contains("/locationMapping") %>'>class="active"</c:if>>
+		<a
+				href="${pageContext.request.contextPath}/module/dhisconnector/locationMapping.form"><spring:message
+				code="dhisconnector.locationMapping" /></a>
+	</li>
+
```

Comment:
Shall we fix the formattings of this section?

Reply:
Alright. I'll update the PR.

Reply:
Fixed with [041fd7b](https://github.com/openmrs/openmrs-module-dhisconnector/commit/041fd7b82d787717170bb3a0acb07c859cc76fd9)